app/models/citation.rb

```diff
@@ -23,4 +23,8 @@ def self.label
     "citation"
   end
 
+  def self.icon
+    "icons/citation.svg"
```

Comment:
Needed to precompile:assets. Is that intended?

Reply:
I didn't; are you using `bin/dev`? In any case, it'll happen in production: https://github.com/xronos-ch/xronos.rails/blob/master/entrypoint.sh